src/js/sdk.tsx

```diff
@@ -50,59 +50,64 @@ export function init(passedOptions: ReactNativeOptions): void {
     stackParser: stackParserFromStackParserOptions(passedOptions.stackParser || defaultStackParser)
   };
 
+  function addIntegration(integration: Integration): void {
```

Comment:
Checking each integration has runtime cost but the list of integrations should max just a few items.

Reply:
We recently talked about this in dotnet. This could technically be a breaking change if one depends o n adding the same integration twice. E.g FileWatcher("path1") and another for path2.

Not a risk here?

Reply:
No, because this is only for adding our own integrations, but actually Sentry JS SDK already deduplicates the integration list, that's under the RN SDK control tho.